flow/ThreadSafeQueue.h

```diff
@@ -27,63 +27,88 @@ THIS SOFTWARE IS PROVIDED BY DMITRY VYUKOV "AS IS" AND ANY EXPRESS OR IMPLIED WA
 
 The views and conclusions contained in the software and documentation are those of the authors and should not be interpreted as representing official policies, either expressed or implied, of Dmitry Vyukov.*/
 
+#include <atomic>
+
```

Comment:
```suggestion
  
  #if VALGRIND
  #include <drd.h>
  #endif
  
```

Reply:
We should really go make a flow/annotations.h sometime that defines these to nothing if valgrind is disabled, and then is also the centralized place to make the "if valgrind use CHECK_MEM_IS_DEFINED else if asan use ASAN_MEM_IS_DEFINED" or whatever.

But that time is not now.